doc/source/_templates/indexcontent.html

```diff
@@ -6,8 +6,8 @@
 {% block body %}
 <h1>{{ docstitle|e }}</h1>
 <p>
-  Welcome! This is the documentation for NumPy {{ release|e }}
-  {%- if last_updated %}, last updated {{ last_updated|e }}{% endif %}.
+  Welcome! This is the documentation for NumPy {{ release|e }} 
+  {%- if last_updated %}, last updated {{ last_updated|e }}{% endif %}. For a PDF version of the documentation, go to https://numpy.org/doc/.
```

Comment:
To make this a clickable link, you can do
```suggestion
  {%- if last_updated %}, last updated {{ last_updated|e }}{% endif %}. For a PDF version of the documentation, go to <a href="https://numpy.org/doc/">https://numpy.org/doc/</a>.
```

Reply:
Thanks, @melissawm, I have made the changes please check.